common/src/main/java/com/microsoft/identity/common/internal/request/MsalBrokerRequestAdapter.java

```diff
@@ -475,29 +499,44 @@ private Bundle getRequestBundleFromBrokerRequest(@NonNull BrokerRequest brokerRe
                     sRequestAdapterGsonInstance.toJson(brokerRequest, BrokerRequest.class)
             );
         }
+        requestBundle.putString(
+                AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY,
+                negotiatedBrokerProtocolVersion
+        );
         return requestBundle;
     }
 
-    public Bundle getRequestBundleForGetAccounts(@NonNull final CommandParameters parameters) {
+    public Bundle getRequestBundleForGetAccounts(@NonNull final CommandParameters parameters,
+                                                 @Nullable final String negotiatedBrokerProtocolVersion) {
         final Bundle requestBundle = new Bundle();
         requestBundle.putString(ACCOUNT_CLIENTID_KEY, parameters.getClientId());
         requestBundle.putString(ACCOUNT_REDIRECT, parameters.getRedirectUri());
+        requestBundle.putString(
+                AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY,
+                negotiatedBrokerProtocolVersion
+        );
         //Disable the environment and tenantID. Just return all accounts belong to this clientID.
         return requestBundle;
     }
 
-    public Bundle getRequestBundleForRemoveAccount(@NonNull final RemoveAccountCommandParameters parameters) {
+    public Bundle getRequestBundleForRemoveAccount(@NonNull final RemoveAccountCommandParameters parameters,
+                                                   @Nullable final String negotiatedBrokerProtocolVersion) {
         final Bundle requestBundle = new Bundle();
         if (null != parameters.getAccount()) {
             requestBundle.putString(ACCOUNT_CLIENTID_KEY, parameters.getClientId());
             requestBundle.putString(ENVIRONMENT, parameters.getAccount().getEnvironment());
             requestBundle.putString(ACCOUNT_HOME_ACCOUNT_ID, parameters.getAccount().getHomeAccountId());
         }
+        requestBundle.putString(
+                AuthenticationConstants.Broker.NEGOTIATED_BP_VERSION_KEY,
+                negotiatedBrokerProtocolVersion
+        );
 
         return requestBundle;
     }
 
-    public Bundle getRequestBundleForRemoveAccountFromSharedDevice(@NonNull final RemoveAccountCommandParameters parameters) {
+    public Bundle getRequestBundleForRemoveAccountFromSharedDevice(@NonNull final RemoveAccountCommandParameters parameters,
```

Comment:
Should we document this public method?

Reply:
Addressed via https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/918/commits/477877f2d99d6eb7d752f18d941a682b51665c62